concrete-core-bench/src/core.rs

```diff
@@ -31,6 +31,14 @@ macro_rules! bench {
     };
 }
 
+// Helper aliases for view fixtures which require knowing what the container type is
+type Vecu32 = Vec<u32>;
+type Vecu64 = Vec<u64>;
+type Sliceu32 = &'static [u32];
+type Sliceu64 = &'static [u64];
+type MutSliceu32 = &'static mut [u32];
+type MutSliceu64 = &'static mut [u64];
```

Comment:
Maybe here we can drop the `u` and keep just Vec32, etc.? Then the shortened names would look nicer. :slightly_smiling_face:

Reply:
Yep for sure, I'll do that !

Reply:
done !